test/integration/test_search.rb

```diff
@@ -26,18 +26,6 @@
 
     search_term = "#find-me"
 
-    create_page name: "Result 1", content: search_term
-    create_page name: "Result 2", content: search_term
-    create_page name: "Result 3", content: search_term
-    create_page name: "Result 4", content: search_term
-    create_page name: "Result 5", content: search_term
-    create_page name: "Result 6", content: search_term
-    create_page name: "Result 7", content: search_term
-    create_page name: "Result 8", content: search_term
-    create_page name: "Result 9", content: search_term
-    create_page name: "Result 10", content: search_term
-    create_page name: "Result 11", content: search_term
-
     fill_in "Search site", with: search_term
     send_keys :enter
```

Comment:
Thanks for the feedback @benjaminwil, I should perhaps not have merged this so hastily and with so little explanation.

I actually agree with you testing-philosophy-wize here, both in terms of expressiveness and in terms of it being good to test page creation. The issue was just that I noticed tests were failing on the "Build and Test Docker" task, seemingly because the app had trouble keeping up with how fast Capybara was trying to save and then click the `New` button. Making sure the pages with search results were already in the wiki was the only solution I came up with in the moment. However, perhaps a better fix would have been to build in some waiting condition. Next time I'll ask for feedback before proceeding!

What do you think would be the best way to proceed now? Shall we try to restore the old test and add a waiting condition?

Reply:
On rereading the tests, I think I actually made a mistake when adding them.

After `click_on 'Save'` I never make any assertions that the page has been successfully created. I think any assertion we would make about content on the new page would force Capybara to wait for its `default_max_wait_time`, which would resolve the timing issues _I would hope_. 

Sorry for making this mistake. If we leave these tests as merged, I think this would be okay. But in the future I will try to remember this about Capybara.

Reply:
Ah thanks for that explanation! That helps me to understand better what was going on, and generally how Capybara works.